ledger/participant-integration-api/src/main/scala/platform/configuration/InitialLedgerConfiguration.scala

```diff
@@ -8,9 +8,9 @@ import com.daml.ledger.configuration.{Configuration, LedgerTimeModel}
 
 /** Instructions on how to initialize an empty ledger, without a configuration.
   *
-  * A configuation is only submitted if one is not detected on the ledger.
+  * A configuration is only submitted if one is not detected on the ledger and `enabled` flag is set to `true`
   *
-  * @param generation                The configuration generation. Monotonically increasing.
+  * @param enabled                Flag which defines if this functionality is enabled.
```

Comment:
This looks strange provided that `enabled` is actually no a param

Reply:
I was adding the flag, and then removing, but forgot to remove it from the doc :)